web/packages/teleport/src/Discover/Database/EnrollRdsDatabase/RdsDatabaseList.tsx

```diff
@@ -16,11 +16,16 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import { Flex, Box, Label as Pill } from 'design';
-import Table, { Cell as TableCell } from 'design/DataTable';
+import { Flex, Box } from 'design';
+import Table from 'design/DataTable';
 import { FetchStatus } from 'design/DataTable/types';
 
-import { Label } from 'teleport/types';
+import {
+  DisableableCell as Cell,
```

Comment:
This was what I initially named it though I think `DisableableCell` is more correct, since it just means a Cell that can be disabled, whereas `DisabledCell` sounds like it's already always disabled